NSSNamespace/Initial_Bangazon_Site
# Jlpayment ctrl

Details:

1. Customer can now create a payment type and save it in the database
2. Order/cart view now contains dropdown of payment types associated with the active customer


To Test:

1. Git fetch
2. Git pull origin jlpaymentCtrl
3. Select an active customer from the dropdown
4. Select the cart icon below that customer's name
5. Add a payment type and ensure it populates in your database
6. Ensure that the active customer's associated payment types populate in the dropdown
